tests/Moq.Tests/Regressions/IssueReportsFixture.cs

```diff
@@ -2010,6 +2011,32 @@ public interface IMyInterface
 
 		#endregion
 
+		#region 706
+		
+		public class Issue706
+		{
+			[Fact]
+			public void CallBase_should_not_be_allowed_for_delegate_mocks()
+			{
+				Mock<Action> mock = new Mock<Action>();
+				Language.Flow.ISetup<Action> setup = mock.Setup(m => m());
+				
+				Exception ex = Assert.Throws<NotSupportedException>(() => setup.CallBase());
+				Assert.Equal(string.Format(CultureInfo.CurrentCulture, Resources.CallBaseUsedOnDelegateException), ex.Message);
+			}
+
+			[Fact]
+			public void CallBase_property_should_not_be_allowed_for_delegate_mocks()
+			{
+				Mock<Action> mock = new Mock<Action>();
+
+				Exception ex = Assert.Throws<NotSupportedException>(() => mock.CallBase = true);
+				Assert.Equal(string.Format(CultureInfo.CurrentCulture, Resources.CallBaseUsedOnDelegateException), ex.Message);
+			}
```

Comment:
Nice! :+1: Could you please add a third test, verifying that `mock.CallBase = false` does *not* throw? Thank you!